README.md

```diff
@@ -13,25 +13,29 @@ npm install @edgepi-cloud/node-red-edgepi-led
 **RPC Server** <br>
 The connection to your EdgePi's RPC Server.
 
-**Configure with**<br>
-Whether to configure LEDs based on configurations from the editor or from input received in the flow.
-
 ## Inputs
 
-- When configuration is set to the editor:
-  Any message can be used to trigger this node.
+  - **payload** (*string*)<br>
+  The LED state.
+  - **pin** (*JSON* `{"argName":"argValue"..}`)<br>
+The LED pin on which to change state.
```

Comment:
why payload is string here?

I don't understand what `{"argName":"argValue"..}` is supposed to mean? isn't pin just an integer?

Reply:
Forgot to change the type from previous version. Fixed.